rules/builtins.build_defs

```diff
@@ -12,7 +12,8 @@ def build_rule(name:str, cmd:str|dict='', test_cmd:str|dict='', srcs:list|dict=N
                licences:list=CONFIG.DEFAULT_LICENCES, test_outputs:list=None, system_srcs:list=None, stamp:bool=False,
                tag:str='', optional_outs:list=None, progress:bool=False, size:str=None, _urls:list=None,
                internal_deps:list=None, pass_env:list=None, local:bool=False, output_dirs:list=[], __=None,
-               exit_on_error:bool=CONFIG.EXIT_ON_ERROR, entry_points:dict={}, env:dict={}, _file_content:str=None):
+               exit_on_error:bool=CONFIG.EXIT_ON_ERROR, entry_points:dict={}, env:dict={}, _file_content:str=None,
+               remote_platform:dict=None):
```

Comment:
Did you consider making this a special label? There's a few cases where Please looks at the labels on build rules and this seems like it would fit. `labels = ["remote-platform:OSFamily=linux", "remote-platform:WorkerType=large"]`.

Reply:
I did. Requires a bit more parsing later though and I thought you'd like the safety ;-)

Also happy with that though since it's pretty esoteric and I'm not sure every aspect of remote execution deserves a first-class field.